### PR TITLE
DOC: reformat doc groupby.rst

### PR DESCRIPTION
**Pull request by erfannariman**
ref #36777

### CODE REVIEW

File: doc/source/user_guide/groupby.rst [hunk 1]
```diff
@@ -659,9 +675,11 @@ and unpack the keyword arguments
 
 .. ipython:: python
 
-   animals.groupby("kind").agg(**{
-       'total weight': pd.NamedAgg(column='weight', aggfunc=sum),
-   })
+   animals.groupby("kind").agg(
+       **{
+           "total weight": pd.NamedAgg(column="weight", aggfunc=sum),
```
Comment (dsaxton) [inline doc/source/user_guide/groupby.rst:680]:
```suggestion
           "total weight": pd.NamedAgg(column="weight", aggfunc=sum)
```

**Review by dsaxton — APPROVED** [review]
Tiny nitpick, otherwise looks good

**Comment by erfannariman** [comment]
Apparently this was already fixed in another PR, now the only change is the removal of the comma @dsaxton

**Comment by dsaxton** [comment]
Thanks @erfannariman

**Pull request merged**